recipes/hyperscan/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conans import ConanFile, CMake, tools
+
+import os
+
+class HyperscanConan(ConanFile):
+    name = "hyperscan"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.hyperscan.io"
+    description = "High-performance regular expression matching library"
+    topics = ("regex", "regular expressions")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+
+    _cmake = None
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "optimise": [True, False],
+        "debug_output": [True, False],
+        "build_avx512": [True, False],
+        "fat_runtime": [True, False],
+        "build_chimera": [True, False],
+        "dump_support": [True, False]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "optimise": True,
+        "debug_output": False,
+        "build_avx512": False,
+        "fat_runtime": False,
+        "build_chimera": False,
+        "dump_support": False
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("hyperscan-{0}".format(self.version), self._source_subfolder)
+
+    def build_requirements(self):
+        self.build_requires("boost/1.75.0");
```

Comment:
Pretty sure it's a requirement, not a build requirement.

Reply:
I'd have to check in detail but I'm confident that all boost libs that are used by hyperscan are header-only. At least for me locally (Arch Linux) I can confirm that boost is only a make dependency.

Reply:
The purpose of build requirements is not to model private dependency, but requirements on the build machine (usually executables run at build time).
So `boost` should be a requirement.